src/test/java/com/github/tomakehurst/wiremock/matching/MatchesJsonPathPatternTest.java

```diff
@@ -456,6 +459,48 @@ public void matchesCorrectlyWhenSubMatcherIsUsedAndExpressionReturnsASingleItemA
     assertTrue(result.isExactMatch());
   }
 
+  public void matchesCorrectlyWhenUsingActualFormatWithJustTheDayPartOfTheDateAndTruncatingTheDateToo() {
+  public void
```

Comment:
Pretty sure this shouldn't be here

Reply:
Yep. Looks  like spotless went wrong inhttps://github.com/wiremock/wiremock/pull/1710/commits/df792b7321010788deddc186140f35642230903b